README.md

```diff
@@ -52,6 +52,16 @@ To automatically add a Task Failure Handler to each `Task` state that will trigg
 }
 ```
 
+To use serverless unpacking for nonstring values, use the `render_nonstring_value()` flag in your `playbook.json`. The example below is referencing an Int type in the serverless.yml custom variables definition:
```

Comment:
can we perfiix this to `apb_render_nonstring_value`? Primary concern here is if we accidentally add a `render_nonstring_value` function later down the line in SOCless via Jinja templates, we'd end up with conflicting behaviour. Unlikely but could happen